search/script.js

```diff
@@ -0,0 +1,23 @@
+var a = [0, 1, 2, 2, 3, 4, 11, 22, 69, 235, 459, 637, 789, 854];
+
+console.log(recursive(a, 235));
+console.log(recursive(a, 0));
+console.log(recursive(a, 854));
```

Comment:
could you try to call the same but when `a = [0, 1, 2, 2, 3, 4, 11, 22, 69, 235, 459, 637];`
looks like it returns index of last element

or with `a = [];`
all calls returns 0